src/components/Header/index.tsx

```diff
@@ -59,107 +59,114 @@ export function useHeaderHeight(): number {
 
 const MenuPopover: FC = () => {
   const [t] = useTranslation('common', { keyPrefix: 'navigation' })
+  const [openMenuPanel, setOpenMenuPanel] = useState<boolean>(false)
+
   return (
-    <Popover className={styles.menuPopover}>
-      {({ close }) => (
-        <>
-          <Popover.Button as={'div'} className={clsx(styles.trigger, DISABLE_CGOL_MOUSE_CONTROLLER)}>
-            <MenuIcon />
-            <span className={styles.text}>MENU</span>
-          </Popover.Button>
+    <div className={styles.menuPopover}>
```

Comment:
A comment is needed here to explain why `Popover` is not used, otherwise it may be switched back to use `Popover` component in the future.

Also I can open it normally on Android phone, maybe it's only an issue on iOS.